pages/blogs/search.js

```diff
@@ -7,28 +7,27 @@ import matter from 'gray-matter';
 import ContentBox from '@components/ContentBox';
 import Search from '@components/Search';
 
-export default ({ allMdx }) => {
+export default function SearchPage({ allMdx }) {
   const [filteredBlogs, setFilteredBlogs] = React.useState(allMdx);
 
   const handleFilter = (data) => {
     setFilteredBlogs(data);
   };
 
   return (
-    <Flex>
+    <Box pb={3}>
```

Comment:
`Flex` was causing the container to not take up the full width, and it's not needed anyway. Bottom padding makes it so the page doesn't end abruptly at the bottom border of the last content box.